.travis.yml

```diff
@@ -0,0 +1,45 @@
+---
```

Comment:
we should be able to avoid adding the travis/appveyor files by configuring `.sync.yml` appropriately - see https://github.com/puppetlabs/puppetlabs-scheduled_task/blob/main/.sync.yml for an example